src/App.css

```diff
@@ -62,3 +55,30 @@ body {
 .zero {
   grid-column: 1 / span 2;
 }
+
+.li-navbar {
+  list-style: none;
+  padding-inline: 0.5rem;
+}
+
+.li-mid {
+  border-right: 1px solid black;
+  border-left: 1px solid black;
+}
+
+h1 {
+  font-size: 2em;
+  margin: 0;
+}
+
+.cal-cont {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem;
+}
```

Comment:
- [ ] [OPTIONAL] Great implementation @Alexr16 , however, the UI could be made better by placing the currently at the center of the page by aligning and justifying it to the center.

### Your Implementation
![Screenshot from 2022-04-26 21-56-58](https://user-images.githubusercontent.com/58692590/165391434-4abb0bd1-d790-49db-af35-a7dcb8ed103d.png)

Reply:
@Whoistolu I'll work on that.
Thanks for your recommendation.